lib/xls_function/evaluators/functions/switch.rb

```diff
@@ -0,0 +1,58 @@
+module XlsFunction
+  module Evaluators
+    module Functions
+      class Switch < ::XlsFunction::Evaluators::FunctionEvaluator
+        function_as :switch
+
+        MAX_ARGUMENTS_COUNT = 254
```

Comment:
`MAX_ARGUMENTS_COUNT` is total value below.
- Value to switch (1)
- What value do you want to match (126 * 2 = 252)
- Default value to return if there's no match found (1)